### PR TITLE
export PAPERLESS_REDIS for apps that need it

### LINKED ISSUE #18: specifying external redis does not work
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

## Expected Behavior
Setting `REDIS_URL=redis://myredis` should connect to my external redis instance and boot the app successfully.

## Current Behavior
The applications try to connect to `redis://localhost:6379` and fail.

## Steps to Reproduce

```sh
docker network create redis-test
docker run --name myredis --net redis-test -d redis
docker run -it --rm -e 'REDIS_URL=redis://myredis' --net redis-test linuxserver/paperless-ng
``` 

## Environment
**OS:** flatcar linux
**CPU architecture:** x86_64
**How docker service was installed:** native

## Docker logs

```
19:46:19 [Q] ERROR Can not connect to Redis server.
Process Process-1:
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/redis/connection.py", line 559, in connect
    sock = self._connect()
  File "/usr/local/lib/python3.8/dist-packages/redis/connection.py", line 615, in _connect
    raise err
  File "/usr/local/lib/python3.8/dist-packages/redis/connection.py", line 603, in _connect
    sock.connect(socket_address)
OSError: [Errno 99] Cannot assign requested address

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/usr/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.8/dist-packages/django_q/cluster.py", line 168, in __init__
    self.start()
  File "/usr/local/lib/python3.8/dist-packages/django_q/cluster.py", line 171, in start
    self.broker.ping()
  File "/usr/local/lib/python3.8/dist-packages/django_q/brokers/redis_broker.py", line 39, in ping
    raise e
  File "/usr/local/lib/python3.8/dist-packages/django_q/brokers/redis_broker.py", line 36, in ping
    return self.connection.ping()
  File "/usr/local/lib/python3.8/dist-packages/redis/client.py", line 1378, in ping
    return self.execute_command('PING')
  File "/usr/local/lib/python3.8/dist-packages/redis/client.py", line 898, in execute_command
    conn = self.connection or pool.get_connection(command_name, **options)
  File "/usr/local/lib/python3.8/dist-packages/redis/connection.py", line 1192, in get_connection
    connection.connect()
  File "/usr/local/lib/python3.8/dist-packages/redis/connection.py", line 563, in connect
    raise ConnectionError(self._error_message(e))
redis.exceptions.ConnectionError: Error 99 connecting to localhost:6379. Cannot assign requested address.
```

### PR DESCRIPTION
**Pull request by chasebolt**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/docker-paperless-ng/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

## Description:
export `PAPERLESS_REDIS` for the apps that need it. include working examples for the `REDIS_URL`.

## Benefits of this PR and context:
fixes #18

## How Has This Been Tested?
built a new docker container based off of `FROM linuxserver/paperless-ng` with my changes. tested the reproducible steps in #18 was resolved.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ Container images are configured using parameters passed at runtime (such as thos
 | `-e PUID=1000` | for UserID - see below for explanation |
 | `-e PGID=1000` | for GroupID - see below for explanation |
 | `-e TZ=America/New_York` | Specify a timezone to use EG America/New_York |
-| `-e REDIS_URL=` | Specify an external redis instance to use. Can optionally include a port (`redis:6379`) and/or db (`redis/foo`). If left blank or not included, will use a built-in redis instance. If changed after initial setup will also require manual modification of /config/settings.py |
```
Comment (Roxedus) [inline README.md:119]: This is not how to edit the readme. You need to change the readme-vars.yml

File: root/etc/services.d/consumer/run [hunk 2]
```diff
@@ -10,4 +12,3 @@ fi
 
 exec \
     s6-setuidgid abc python3 manage.py document_consumer
-
```
Comment (Roxedus) [inline root/etc/services.d/consumer/run:13]: Please keep this newline, across all the runfiles

**Review by Roxedus — CHANGES_REQUESTED** [review]

File: root/etc/services.d/consumer/run (hunk 2 above)
Comment (chasebolt) [reply root/etc/services.d/consumer/run:13]: this newline isnt in the paperless runfile (https://github.com/linuxserver/docker-paperless-ng/blob/master/root/etc/services.d/paperless/run) and it is actually 2 newlines which I don't think is correct?

File: README.md (hunk 1 above)
Comment (chasebolt) [reply README.md]: doh! should have read the top of the readme - I just did ctrl+f for redis. fixed now.

File: root/etc/services.d/consumer/run (hunk 2 above)
Comment (chasebolt) [reply root/etc/services.d/consumer/run:13]: https://i.imgur.com/uHcNPgX.png

Comment (chasebolt) [reply root/etc/services.d/consumer/run:13]: @Roxedus what is your verdict on this? should I put the double newlines back in?

**Pull request closed without merging**